clodius/tiles/multivec.py

```diff
@@ -284,6 +285,9 @@ def tileset_info(filename):
     if "row_infos" in f["resolutions"][str(resolutions[0])].attrs:
         row_infos = f["resolutions"][str(resolutions[0])].attrs["row_infos"]
         tileset_info["row_infos"] = [r.decode("utf8") for r in row_infos]
+    elif "row_infos" in f["info"].keys():
```

Comment:
You don't need the `.keys()` here. `"row_infos" in f["info"]` works the same :-)